### PR TITLE
[Table] Reduce shadow intensity

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

Reduce sticky table shadow intensity

-----

image

image

-----

### CODE REVIEW

File: packages/scss/src/components/tableSticked/vars.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
 @mixin vars {
-	--components-tableSticked-column-sticky-shadow-width: 0.625rem;
-	--components-tableSticked-column-sticky-shadow-color: rgba(42, 53, 81, 0.25);
+	--components-tableSticked-column-sticky-shadow-width: 0.5rem;
+	--components-tableSticked-column-sticky-shadow-color: rgba(172, 187, 215, 0.25);
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/tableSticked/vars.scss:3]: color mix + css var ?

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2824/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2824/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17611/ui-diff)

**Review by BertrandPodevin — APPROVED** [review]

**Pull request merged**